source/frontend/src/constants/documentType.ts

```diff
@@ -0,0 +1,3 @@
+export enum DocumentTypeName {
+  CDOGS = 'CDOGS Template',
```

Comment:
My fault on this one, one of my stories changed CDOGS Template to CDOGS template.